generators/app/templates/Dockerfile

```
@@ -16,19 +19,27 @@ RUN dotnet build
 
 WORKDIR /app/test
 RUN dotnet test StarterKit.UnitTests
+RUN dotnet test StarterKit.IntegrationTests
 
 # publish
 WORKDIR /app/src/StarterKit
 
 RUN dotnet publish -c Release -o out
 
 # Build runtime image
-FROM mcr.microsoft.com/dotnet/core/aspnet:2.2.5 AS runtime
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.5-focal AS runtime
 WORKDIR /app
 COPY --from=build-env /app/src/StarterKit/out .
```

Dit moet volgens mij het volgende worden:

```suggestion
COPY --from=build-env /app/out .
```

De output folder is sinds .NET Core 3.x SDK nl. relatief aan de working folder: 

> .NET Core 3.x SDK and later
> 
> If a relative path is specified when publishing a project, the output directory generated is relative to the current working directory, not to the project file location.

https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish#options

---

Deze comment klopt niet. De folder klopt omdat de `publish` al gebeurd in `WORKDIR /app/src/FacturenApi`.

Ik was op de aanpassing van de output folder sinds 3.x gebotst in een andere toepassing, waar de WORKDIR anders was.